windows/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.19-nanoserver AS workspace
+LABEL builder=true
+
+ENV CGO_ENABLED=0
+
+WORKDIR /go/src/github.com/testcontainers/moby-ryuk
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . ./
+RUN cd /go/src/github.com/testcontainers/moby-ryuk && go get -d \
+ && go vet ./... \
+ && go test ./... \
+ && go build \
+    -v -a \
+    -ldflags "-s -w -extldflags \"-static\"" \
+    -o /bin/moby-ryuk main.go
+
+FROM mcr.microsoft.com/windows/nanoserver:ltsc2022
```

Comment:
I think we need to consider different Windows versions. The 2022 base image, is not compatible with every host OS version. This is the [compatibility](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility?tabs=windows-server-2022%2Cwindows-11-21H2) matrix. Either be build different versions or use 2019 as the base version. It looks like it is compatible with the most recent versions.

Reply:
This one might help and even address your other questions below: https://lippertmarkus.com/2021/11/30/win-multiarch-img-lin/

Reply:
That's interesting. I'll take a look at it tomorrow too, thanks.

Reply:
I gave the cross-build a shot on my Windows system, with LCOW [sic], which failed. So, the "cross-build" is to be taken seriously, or in other words: building Windows images that way works on Linux systems, only. I'm confused now 🤔

<img width="952" alt="Screenshot 2022-08-09 222550" src="https://user-images.githubusercontent.com/432791/183754100-aeb4dd0d-d9e0-4a49-b90a-c17d5939d11c.png">

Reply:
Fails on macOS/Docker Desktop, too.
Admittedly, I didn't configure any QEMU or whatever might be required on non-plain Linux systems.